debian/portal.triggers

```diff
@@ -0,0 +1,8 @@
+# Register interest in Python interpreter changes (Python 2 for now); and
+# don't make the Python package dependent on the virtualenv package
+# processing (noawait)
+interest-noawait /usr/bin/python2.6
```

Comment:
Don't bother supporting 2.6 - it's too old and I'm sure we'd run into some syntax errors with the current code base